### PR TITLE
Convert two baremetal nodes to compute

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

**Review by markgoddard — COMMENTED** [review]
You'll need interface configuration in `etc/kayobe/inventory/group_vars/compute/network-interfaces`. See `etc/kayobe/inventory/group_vars/controllers/network-interfaces` for a starting point, but you will only need a subset of that.

You'll also need to update `etc/kayobe/compute.yml`, similar to `etc/kayobe/controllers.yml`. You'll want to add a role mapping for the computes in `etc/kayobe/overcloud.yml`. And you'll want to set `kolla_enable_neutron_provider_networks` to true.

File: etc/kayobe/compute.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 # User with which to access the computes via SSH during bootstrap, in order
 # to setup the Kayobe user account.
 #compute_bootstrap_user:
+compute_bootstrap_user: root
```
Comment (jovial) [inline etc/kayobe/compute.yml]:
```suggestion
compute_bootstrap_user: centos
```
centos is default user for generic cloud image

File: etc/kayobe/users.yml [hunk 2]
```diff
@@ -20,15 +21,8 @@ users_default:
   ssh_key:
   # Authorise all users' SSH keys for the kolla user. This allows them to
   # operate kolla.
-  - "{{ lookup('file', kayobe_config_path ~ '/ssh-keys/id_rsa_bharat.pub') }}"
-  - "{{ lookup('file', kayobe_config_path ~ '/ssh-keys/id_rsa_isaac.pub') }}"
-
-- username: isaac
```
Comment (jovial) [inline etc/kayobe/users.yml:26]: Did you mean to remove @Wasaac's account definition? It is probably better practice not to share accounts. Also this won't delete the account.

Comment (brtkwr) [reply etc/kayobe/users.yml:26]: I thought it might have been added here by accident. Does it really make sense to have an account per user in all the overcloud and seed machines?

File: etc/kayobe/inventory/host_vars/ethsw-b16-u38 [hunk 3]
```diff
@@ -146,6 +146,11 @@ switch_interface_config:
       - "no shutdown"
       # Controllers.
       - "tagged Te 1/5/2,1/5/4"
+  vlan 41:
+    description: overcloud provisioning
+    config:
+      - "no shutdown"
+      - "untagged Te 1/4/4"
```
Comment (markgoddard) [inline etc/kayobe/inventory/host_vars/ethsw-b16-u38]: Why is this required?

File: etc/kayobe/users.yml (hunk 2 above)
Comment (markgoddard) [reply etc/kayobe/users.yml:26]: Anyone who wants to run kayobe needs to have their key authorized for the stack and kolla users.

Comment (brtkwr) [reply etc/kayobe/users.yml:26]: The keys are intact for stack and kolla users

File: etc/kayobe/inventory/host_vars/ethsw-b16-u38 (hunk 3 above)
Comment (jovial) [reply etc/kayobe/inventory/host_vars/ethsw-b16-u38]: This is a relic from when we were going to move the nodes into bifrost. This was before we realised there was no physical connection between the 1G and 10G networks. We should remove this.

**Review by jovial — APPROVED** [review]
This is going to be the basis of the train config, could we merge so that the diff isn't so large? I've removed the contentious switch config.
